.github/workflows/ci.yml

```diff
@@ -66,7 +64,7 @@ jobs:
       matrix:
         TEST:
           - {CONTAINER: "stretch", TOXENV: "py27"}
-          - {CONTAINER: "stretch", TOXENV: "py35"}
+          - {CONTAINER: "ubuntu-bionic", TOXENV: "py36"}
```

Comment:
I tried changing the distro test to ubuntu-bionic (stretch only has 3.5), which fails due to the old pip version + cryptography 3.4. Do you want to update pip here, or specify RUSTUP_HOME as it is done in the cryptography tests [here](https://github.com/pyca/cryptography/blob/5fdc9c1b8563c2ffe99d6dff3f0632fb9dc29936/.github/workflows/ci.yml#L134)?

Reply:
Let's upgrade pip here. No need to compile cryptography if we don't have to...